tests/testthat/_snaps/backend-sqlite.md

```diff
@@ -17,15 +16,24 @@
     Output
       <SQL>
       SELECT `x`, `y.x`, `y.y`, `z`
-      FROM (SELECT COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`, `LHS`.`y` AS `y.x`, `RHS`.`y` AS `y.y`, `z`
-      FROM `df` AS `LHS`
-      LEFT JOIN `df` AS `RHS`
-      ON (`LHS`.`x` = `RHS`.`x`)
-      UNION
-      SELECT COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`, `LHS`.`y` AS `y.x`, `RHS`.`y` AS `y.y`, `z`
-      FROM `df` AS `RHS`
-      LEFT JOIN `df` AS `LHS`
-      ON (`LHS`.`x` = `RHS`.`x`)
+      FROM (
+        SELECT
+          COALESCE(`LHS`.`x`, `RHS`.`x`) AS `x`,
+          `LHS`.`y` AS `y.x`,
+          `RHS`.`y` AS `y.y`,
+          `z`
+        FROM `df` AS `LHS`
+        LEFT JOIN `df` AS `RHS`
+        ON (`LHS`.`x` = `RHS`.`x`)
```

Comment:
I think `ON` should either go on the same line as `LEFT JOIN` or be indented?

Reply:
I indented `ON` because this was a bit easier